### PR TITLE
reflect vote counts in client UI

### PR DESCRIPTION
**Pull request by MauriceNaftalin**

### CODE REVIEW

File: src/main/resources/static/phoneClient.html [hunk 1]
```diff
@@ -56,4 +63,4 @@
 
 </script>
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (lifey) [inline src/main/resources/static/phoneClient.html:75]: 👍

File: src/main/resources/static/phoneClient.html [hunk 2]
```diff
@@ -4,9 +4,12 @@
     <meta charset="UTF-8">
     <title>Phone Client</title>
     <script src="https://cdn.jsdelivr.net/npm/@stomp/stompjs@7.0.0/bundles/stomp.umd.min.js"></script>
+    <script src="https://d3js.org/d3.v7.min.js"></script>
     <script src="/phoneClientStompManager.js"></script>
 </head>
 <body>
+<svg width="250" height="250" id="svg"></svg>
```
Comment (lifey) [inline src/main/resources/static/phoneClient.html]: 250 is 250px i guess maybe you want to make it more dynamic based on the size of the screen (browser window)

**Review by lifey — COMMENTED** [review]
minor comments

File: src/main/resources/static/phoneClient.html (hunk 2 above)
Comment (MauriceNaftalin) [reply src/main/resources/static/phoneClient.html]: "Minor comment"? Perhaps, in well-structured code showing a good understanding of d3. As it is... anyway, works now for phoneClient; the presentation will be able to choose the iframe dimensions too.

**Review by lifey — APPROVED** [review]

**Pull request merged**